webapp/package.json

```diff
@@ -45,7 +45,7 @@
     "@ngx-translate/core": "^13.0.0",
     "@webcomponents/webcomponentsjs": "^2.7.0",
     "bikram-sambat": "^1.7.0",
-    "bikram-sambat-bootstrap": "^1.5.0",
+    "bikram-sambat-bootstrap": "https://gitpkg.now.sh/medic/bikram-sambat/bootstrap",
```

Comment:
I'm not familiar with `gitpkg.now.sh` can we use another option like a git url here instead to avoid relying on this service?

Reply:
I totally agree with this @craig-landry. I initially tried referencing github url, but since this needs to be referenced from subfolder of our repo, referencing that with github url didn't work properly. So, I found Gitpkg, the [open source](https://github.com/EqualMa/gitpkg) tool to reference any github urls, branch, subfolder as a reference. 

I haven't referenced github subfolders previously in npm, if you could provide some working reference or previous examples, then I will update this.

Reply:
This change is temporary, just while this issue is AT-ed. 
After AT, we will publish a new version of `bikram-sambat-bootstrap` on npm, that includes the fix, and PR will be updated to just bump the version to `1.6.0` (instead of directly installing from a branch).

Reply:
> This change is temporary, just while this issue is AT-ed.

Ah, cool. Thanks 👍